src/org/labkey/test/components/ui/entities/EntityBulkInsertDialog.java

```diff
@@ -192,6 +193,18 @@ public boolean getBooleanField(String fieldKey)
         return elementCache().checkBox(fieldKey).get();
     }
 
+    public Optional<WebElement> validationMessage()
```

Comment:
This warning is shown if a test or user attempts to specify more than 1000 things to insert.
`EntityBulkInsertDialog `is the dialog that is shown when the 'bulk insert' button on `EntityInsertPanel `is clicked.